### PR TITLE
Add Theano-pymc master

### PR DESCRIPTION
**Pull request by canyon289**
Trying to fix https://github.com/arviz-devs/arviz/issues/1455

### CODE REVIEW

**Comment by canyon289** [comment]
Hmm Theano is getting installed twice for some reason

https://dev.azure.com/ArviZ/ArviZ/_build/results?buildId=3645&view=logs&j=af9a1d3a-ff8f-5483-6823-86518c773a9b&t=0aaa11b3-1b3c-5ed3-0c54-8fd8e63dd2e8&l=121

**Comment by canyon289** [comment]
Ok found it. Will figure out what to do tomorrow

https://github.com/arviz-devs/arviz/blob/master/.azure-pipelines/azure-pipelines-external.yml#L90

**Comment by ahartikainen** [comment]
Why is it installing it 2 times???

Could this work?:

I think we could add the following env var somewhere in all of our scripts.


    export PIP_IGNORE_INSTALLED=0

Apparently `pip install -r ...` reinstall packages even when they exist already (???)

**Comment by ahartikainen** [comment]
For this 

    python -m pip install --no-cache-dir -r requirements-external.txt

we should replace it with this

    cat requirements-external.txt | grep -E '^pystan|^pyro|^emcee|^pymc3|tensorflow' | xargs python -m pip install

And add explicit install for `tensorflow_probability` when using `pip`

**Comment by canyon289** [comment]
I dont get this comment. Arent we always using pip, so we should install pip so we always need to install it?

`And add explicit install for tensorflow_probability when using pip`

File: .azure-pipelines/azure-pipelines-external.yml [hunk 1]
```diff
@@ -90,7 +90,8 @@ jobs:
           python -m pip --no-cache-dir install pymc3
       fi
 
-      python -m pip install --no-cache-dir -r requirements-external.txt
+      grep -Ev '^pystan|^pyro|^emcee|^pymc3|tensorflow' requirements-external.txt | xargs python -m pip install
```
Comment (ahartikainen) [inline .azure-pipelines/azure-pipelines-external.yml:93]:
```suggestion
      grep -Ev '^pystan|^pyro|^emcee|^pymc3|^tensorflow' requirements-external.txt | xargs python -m pip install
```

File: .azure-pipelines/azure-pipelines-external.yml [hunk 2]
```diff
@@ -90,7 +90,8 @@ jobs:
...
+      python -m pip install tensorflow tensorflow-probability
```
Comment (ahartikainen) [inline .azure-pipelines/azure-pipelines-external.yml:94]:
```suggestion

```

In line 81

```
      if [ "$(tensorflow.version)" = "latest" ]; then
          python -m pip --no-cache-dir install tensorflow tensorflow_probability
      else
          python -m pip --no-cache-dir install tensorflow==1.14 tensorflow_probability==0.7
      fi
```

File: .azure-pipelines/azure-pipelines-external.yml (hunk 1 above)
Comment (ahartikainen) [reply .azure-pipelines/azure-pipelines-external.yml:93]: That `^tensorflow` will take out both tensorflow and tensorflow-prob

**Comment by ahartikainen** [comment]
> I dont get this comment

This is because I usually reply with minimal content 🥇 💯

**Comment by canyon289** [comment]
> > I dont get this comment
> 
> This is because I usually reply with minimal content 🥇 💯

Haha its ok because you wrote the code which is what really matters. Lets see if CI passes now :)

**Comment by ahartikainen** [comment]
`ERROR: Could not find a version that satisfies the requirement tensorflow==1.14
`

Let's change the version

**Comment by canyon289** [comment]
@ahartikainen @ColCarroll if you can review this PR and I will incorporate any additional feedback. For now Im going to merge for now to get back to a green master

**Comment by ahartikainen** [comment]
Looks good.

**Pull request merged**